### ISSUE TITLE
Bottom left to top right stroke of lowercase xi (χ) too thin

### ISSUE THREAD

**Issue by apoorvpotnis** [jarnosz/Concrete-LGC#1]
The bottom left to top right stroke of lowercase xi (χ) seems too thin in comparison to other characters. Is this a bug? 
![Screenshot_20230809_001939](https://github.com/jarnosz/Concrete-LGC/assets/48640612/bc8334a8-264d-4982-8fb2-43002a4767e5)

**Comment by lodwicz** [comment]
not quite a bug; perhaps a misfeature: the program is following the parameters set in cbgreek, with concrete variations: the stroke is meant to be thinner in the original design.

**Comment by apoorvpotnis** [comment]
Hmm. I'll try to mail Claudio Beccari. Can the stroke width be
changed/corrected in this font though?

**Comment by lodwicz** [comment]
Beccari and Knuth are the authors of the Metafont sources, wherefrom they were hacked and mftraced by me; but the font is now in OpenType format; I'd rather modify the glyph in FontForge, or something, if anything...

**Comment by lodwicz** [comment]
I actually like the glyphs as they are, though I know I need to adjust some kernings if I get the time; but you may exercise your right to fork, as usual...

**Comment by lodwicz** [comment]
> Beccari and Knuth are the authors of the Metafont sources, wherefrom they were hacked and mftraced by me; but the font is now in OpenType format; I'd rather modify the glyph in FontForge, or something, if anything...

interested in the metafont source code?

**Comment by apoorvpotnis** [comment]
Yes, but I really don't know how to work with metafont, or know anything
about modifying fonts, in general. But it'll be nice to have the sources.
Thanks.

**Comment by apoorvpotnis** [comment]
Upon closer inspection, it also seems that the outlines are not quite correct, probably an error due to conversion from metafont generated bitmaps. The lines which are supposed to be straight are not quite straight, but rather curves with very low curvature. 

Here is an example of a not quite right serif on right hand lower side.
![Screenshot_20240204_002459](https://github.com/jarnosz/Concrete-LGC/assets/48640612/ed9fb9a5-3d4c-48d7-a6e4-9a9f62619f6c)

Of course these problems will never/rarely be found in regular usage, as one would need to zoom in really hard to notice these things.

**Comment by lodwicz** [comment]
My apologies for the very long delay: I've just posted the source of the Concrete Greek fonts in a related project, for you to inspect...
